endpoints/npm.ts

```diff
@@ -159,3 +164,30 @@ const parseDependents = html => {
   if (depLink.length !== 1) return -1
   return depLink.text().replace(/[^0-9]/g, '')
 }
+
+async function typesDefinition(pkg: string, tag: string = 'latest') {
+    const endpoint = `https://cdn.jsdelivr.net/npm/${ pkg }@${ tag }/package.json`
+    let meta = await got(endpoint).then(res => res.body)
+
+    if (typeof meta.types === 'string' || typeof meta.typings === "string") {
+        return {
+            subject: 'typings',
+            status: 'included',
+            color: 'blue'
```

Comment:
Let's give it a [branding color](https://github.com/microsoft/TypeScript-Website/issues/94):

```suggestion
            color: '3178C6'
```

Reply:
I took it from this issue https://github.com/microsoft/TypeScript-Website/issues/94 TS guys are planning a rebranding.

`#0074c1` is primary color currently 👍